### PR TITLE
fix(transformer): Add getters for `events`.

### LINKED ISSUE #2725: Dart transformer doesn't add getters for `events` in @Component annotations.
If you add the transformer to the zippy_component example you can see this, you will get something like:

Exception: Uncaught Error: Cannot find getter open

### PR DESCRIPTION
**Pull request by jakemac53**
closes https://github.com/angular/angular/issues/2725

[Review on Reviewable](https://reviewable.io/reviews/angular/angular/2729)

### CODE REVIEW

**Comment by jakemac53** [comment]
This doesn't quite get zippy_component working with the transformer, but it does get rid of the first error.

**Comment by jakemac53** [comment]
the travis failure looks like it is in the js code - unrelated to this pr

**Comment by yjbanov** [comment]
We had the build failing temporarily yesterday. You may be on the bad commit. A rebase should fix it.

**Comment by yjbanov** [comment]
LGTM, since we are still generating getters/setters today, but I'd like to see us exit the business of getter/setter generation in favor of generated change detectors. They are untyped and non-minifiable.

**Comment by mhevery** [comment]
@yjbanov, agreed. For that we have to update the code generation to turn statements into methods on ChangeDetector. That is the plan. @vsavkin has to do that work on JS side, before we can do it on Dart side.

File: modules/angular2/src/transform/bind_generator/generator.dart [hunk 1]
```diff
@@ -68,3 +78,37 @@ Map<String, String> _createBindMap(NgDeps ngDeps) {
...
+/// the property names for the event emitters and the values are the event name.
+Map<String, String> _createEventsMap(NgDeps ngDeps) {
+  var visitor = new ExtractNamedExpressionVisitor('events');
+  var bindMap = {};
+  ngDeps.registeredTypes.forEach((RegisteredType t) {
+    visitor.bindConfig.clear();
+    t.annotations.accept(visitor);
+    visitor.bindConfig.forEach((String config) {
+      // See comments for `Directive` in annotations_impl/annotations.ts for
+      // details on how `properties` is specified.
```
Comment (vicb) [inline modules/angular2/src/transform/bind_generator/generator.dart]: `events`

**Comment by vicb** [comment]
please fix the comment before merging

File: modules/angular2/src/transform/bind_generator/generator.dart (hunk 1 above)
Comment (jakemac53) [inline modules/angular2/src/transform/bind_generator/generator.dart]: done

**Comment by jakemac53** [comment]
Got various flakes on travis but all modes passed at least once, so I went ahead and merged.

**Pull request merged**